### DISCUSSION TITLE
[BUG] URL error

### DISCUSSION THREAD

**Discussion by niurencheng** [pycaret/pycaret#1525] [Q&A]
**Describe the bug**

**To Reproduce**

```python
<Paste your code here>
```

**Expected behavior**

**Additional context**

**Versions**
<details>

</details>

![LGOJ2JWO)39GMN2@ZI KXKN](https://user-images.githubusercontent.com/57834780/129194311-926db0f9-9b54-43a3-8848-8c346a7ec58f.png)

**Comment by niurencheng** [comment]
a question   -------URLError

**Comment by wkuopt** [comment]
I was not able to reproduce this error, and this does not seem to be related to pycaret. A similar issue suggested to check Python installation and openssl: https://stackoverflow.com/questions/28376506/urllib-https-request-urlopen-error-unknown-url-type-https
